ACCESSIBILITY.md

```diff
@@ -0,0 +1,100 @@
+
+As a community, we are further committed to ensuring broader accessibility of our resources through using this infrastructure through:
+
+- Choosing sufficient color contrast in our hand-drawn images that make our images accessible in the context of color-blindness
+- Using and employing appropriate and relevant alt text (also known as 'alternative text') for all of our images, charts, and data visualisations (an ongoing project with in the community)
+- Navigation without the need for a mouse or track pad during either the reading process on JupyterBook and/or contributing process on Github
+
+### 2.2 Community Platform Accessibility
+
+_The Turing Way_ is committed to ensuring our accessibility as a community on all the platforms we use.
+
+In our [Community Handbook](https://the-turing-way.netlify.app/community-handbook/accessibility), we aim to describe these practices more specifically and in more detail. But as a baseline, our community is dedicated to enabling access across the various platforms we use through:
+- Providing diverse opportunities and platforms for people to contribute to _The Turing Way_ despite barriers or difficulties associated with one platform or another
+- Setting and embodying standards for writing and contribution that understand the way in which language may exclude as well as include people in data science, research, and other collaborative projects
+
+### 2.3 Community Event Accessibility
+
+As a community, we frequently organise events for a variety of purposes: for coworking, trainings, workshops, panels, conference talks, and more.
+
+In our Community Handbook we describe these practices more specifically and in more detail.
+As a baseline however, we are committed to enabling accessibility through:
+
+- Supplying and recording transcripts for audio and closed captioning of video content
+- Enabling wide access to our program context across a variety of platforms
+- Uploading slides and/or presentation materials on Zenodo or other platforms to enable referencing
+
+### 3.1 How we work with Access-related Feedback
+
+Gathering access-related feedback is integral for ensuring that we can remain accessible as a community. We would like to offer a a few ways of sharing feedback with us, and are currently working to develop the following initiatives:
+
+1. **Feedback form**: Please submit access related through [this form](https://forms.gle/Ngr2eUtQmf7aEeao6). All of the information in this form is processed anonymously, with the option of submitting contact information by choice. 
```

Comment:
I update the embedded link:”... [this form](https://forms.gle/Ngr2eUtQmf7aEeao6) .”

Reply:
I like the idea of specifying the two pints about limiting bandwidth use. Selfishly I like that less reliance on video makes things more accessible for screen reader users :) 

I'm not sure if all these specifics, including the details about contrast ratio, should be in the policy or should be with the moe technical content that we plan on developing about web and event accessibility.  Maybe we dhould have a chat about how much detail should be in the plicy and how it should link to more detailed references.

Reply:
This sounds like something that would be better suited for our documentation - definitely important to flag though!